### PR TITLE
TRUS-3575 Basic Trusts-obliged-entity-stub to allow config to be done

### PR DESCRIPTION
**Pull request by carrielies**
- Basic Trusts-obliged-entity-stub to allow config to be done
- Will finish tests once it's in place

### CODE REVIEW

File: app/controllers/ObligedEntitiesController.scala [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+@Singleton()
+class ObligedEntitiesController @Inject()(headerValidator: HeaderValidatorAction)(cc: ControllerComponents)
+    extends BackendController(cc) with HeaderValidator {
+
+  def getObligedEntity(utr: String): Action[AnyContent] = headerValidator.async { implicit request =>
+    if(isUtrValid(utr)) {
+      utr match {
+        //sample
+        case "2134514321" =>
```
Comment (jassalrichy) [inline app/controllers/ObligedEntitiesController.scala:35]: is 2134514321 correct ?
Given I request a record for UTR at /trusts/obliged-entities/UTR/1234567890

File: app/utils/DesResponse.scala [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+       |}""".stripMargin)
+
+  val jsonResponse503: JsValue = Json.parse(
+    s"""
+       |{
+       | "code": "SERVICE_UNAVAILABLE",
+       | "reason": "Dependent systems are currently not responding"
+       |}""".stripMargin)
+
+  val jsonResponse500: JsValue = Json.parse(
```
Comment (adamconder) [inline app/utils/DesResponse.scala:38]: @carrielies These error responses do not match the YAML specification. For example the structure in the YAML is
```
{
  "failures": [
    {
      "code": "INVALID_IDTYPE",
      "reason": "Submission has not passed validation. Invalid parameter idType."
    }
  ]
}
```

File: app/controllers/ObligedEntitiesController.scala (hunk 1 above)
Comment (carrielies) [reply app/controllers/ObligedEntitiesController.scala:35]: That's the test one that came with the documentation of the api. So I believe it's correct

File: app/utils/DesResponse.scala (hunk 2 above)
Comment (carrielies) [reply app/utils/DesResponse.scala:38]: These are already fixed in my latest code. Just going through the yaml now. As mentioned I was just pushing up a quick version of the stub. I'll wait until it's complete though

**Review by adamconder — APPROVED** [review]

**Pull request merged**